2020-08-01 22:25:24,205 ERROR 12749 [egg-mongoose] mongodb://127.0.0.1:27017/interview disconnected
2020-08-01 22:25:24,207 ERROR 12749 nodejs.MongoNetworkError: [egg-mongoose]failed to connect to server [127.0.0.1:27017] on first connect [MongoNetworkError: connect ECONNREFUSED 127.0.0.1:27017]
    at Pool.<anonymous> (/Users/kuban/Desktop/InterviewQuestionManage/node_modules/mongodb/lib/core/topologies/server.js:433:11)
    at Pool.emit (events.js:189:13)
    at createConnection (/Users/kuban/Desktop/InterviewQuestionManage/node_modules/mongodb/lib/core/connection/pool.js:562:14)
    at connect (/Users/kuban/Desktop/InterviewQuestionManage/node_modules/mongodb/lib/core/connection/pool.js:985:11)
    at makeConnection (/Users/kuban/Desktop/InterviewQuestionManage/node_modules/mongodb/lib/core/connection/connect.js:40:11)
    at callback (/Users/kuban/Desktop/InterviewQuestionManage/node_modules/mongodb/lib/core/connection/connect.js:262:5)
    at Socket.err (/Users/kuban/Desktop/InterviewQuestionManage/node_modules/mongodb/lib/core/connection/connect.js:287:7)
    at Object.onceWrapper (events.js:277:13)
    at Socket.emit (events.js:189:13)
    at emitErrorNT (internal/streams/destroy.js:82:8)
    at emitErrorAndCloseNT (internal/streams/destroy.js:50:3)
    at process._tickCallback (internal/process/next_tick.js:63:19)
name: "MongoNetworkError"
errorLabels: ["TransientTransactionError"]
pid: 12749
hostname: senge.local

2020-08-01 22:25:24,208 ERROR 12749 nodejs.MongoNetworkError: [egg-mongoose]failed to connect to server [127.0.0.1:27017] on first connect [MongoNetworkError: connect ECONNREFUSED 127.0.0.1:27017]
    at Pool.<anonymous> (/Users/kuban/Desktop/InterviewQuestionManage/node_modules/mongodb/lib/core/topologies/server.js:433:11)
    at Pool.emit (events.js:189:13)
    at createConnection (/Users/kuban/Desktop/InterviewQuestionManage/node_modules/mongodb/lib/core/connection/pool.js:562:14)
    at connect (/Users/kuban/Desktop/InterviewQuestionManage/node_modules/mongodb/lib/core/connection/pool.js:985:11)
    at makeConnection (/Users/kuban/Desktop/InterviewQuestionManage/node_modules/mongodb/lib/core/connection/connect.js:40:11)
    at callback (/Users/kuban/Desktop/InterviewQuestionManage/node_modules/mongodb/lib/core/connection/connect.js:262:5)
    at Socket.err (/Users/kuban/Desktop/InterviewQuestionManage/node_modules/mongodb/lib/core/connection/connect.js:287:7)
    at Object.onceWrapper (events.js:277:13)
    at Socket.emit (events.js:189:13)
    at emitErrorNT (internal/streams/destroy.js:82:8)
    at emitErrorAndCloseNT (internal/streams/destroy.js:50:3)
    at process._tickCallback (internal/process/next_tick.js:63:19)
name: "MongoNetworkError"
errorLabels: ["TransientTransactionError"]
pid: 12749
hostname: senge.local

2020-08-01 22:25:24,210 ERROR 12749 [-/127.0.0.1/-/0ms GET /] nodejs.MongoNetworkError: [egg-mongoose]failed to connect to server [127.0.0.1:27017] on first connect [MongoNetworkError: connect ECONNREFUSED 127.0.0.1:27017]
    at Pool.<anonymous> (/Users/kuban/Desktop/InterviewQuestionManage/node_modules/mongodb/lib/core/topologies/server.js:433:11)
    at Pool.emit (events.js:189:13)
    at createConnection (/Users/kuban/Desktop/InterviewQuestionManage/node_modules/mongodb/lib/core/connection/pool.js:562:14)
    at connect (/Users/kuban/Desktop/InterviewQuestionManage/node_modules/mongodb/lib/core/connection/pool.js:985:11)
    at makeConnection (/Users/kuban/Desktop/InterviewQuestionManage/node_modules/mongodb/lib/core/connection/connect.js:40:11)
    at callback (/Users/kuban/Desktop/InterviewQuestionManage/node_modules/mongodb/lib/core/connection/connect.js:262:5)
    at Socket.err (/Users/kuban/Desktop/InterviewQuestionManage/node_modules/mongodb/lib/core/connection/connect.js:287:7)
    at Object.onceWrapper (events.js:277:13)
    at Socket.emit (events.js:189:13)
    at emitErrorNT (internal/streams/destroy.js:82:8)
    at emitErrorAndCloseNT (internal/streams/destroy.js:50:3)
    at process._tickCallback (internal/process/next_tick.js:63:19)
name: "MongoNetworkError"
errorLabels: ["TransientTransactionError"]
pid: 12749
hostname: senge.local

2020-08-01 23:24:18,329 ERROR 12818 nodejs.ClusterClientNoResponseError: client no response in 900747ms exceeding maxIdleTime 60000ms, maybe the connection is close on other side.
    at Timeout.Leader._heartbeatTimer.setInterval [as _onTimeout] (/Users/kuban/Desktop/InterviewQuestionManage/node_modules/cluster-client/lib/leader.js:77:23)
    at ontimeout (timers.js:436:11)
    at tryOnTimeout (timers.js:300:5)
    at listOnTimeout (timers.js:263:5)
    at Timer.processTimers (timers.js:223:10)
name: "ClusterClientNoResponseError"
pid: 12818
hostname: senge.local

2020-08-01 23:57:02,540 ERROR 12818 nodejs.ClusterClientNoResponseError: client no response in 1923930ms exceeding maxIdleTime 60000ms, maybe the connection is close on other side.
    at Timeout.Leader._heartbeatTimer.setInterval [as _onTimeout] (/Users/kuban/Desktop/InterviewQuestionManage/node_modules/cluster-client/lib/leader.js:77:23)
    at ontimeout (timers.js:436:11)
    at tryOnTimeout (timers.js:300:5)
    at listOnTimeout (timers.js:263:5)
    at Timer.processTimers (timers.js:223:10)
name: "ClusterClientNoResponseError"
pid: 12818
hostname: senge.local

2020-08-02 00:29:41,354 ERROR 12818 nodejs.ClusterClientNoResponseError: client no response in 1918250ms exceeding maxIdleTime 60000ms, maybe the connection is close on other side.
    at Timeout.Leader._heartbeatTimer.setInterval [as _onTimeout] (/Users/kuban/Desktop/InterviewQuestionManage/node_modules/cluster-client/lib/leader.js:77:23)
    at ontimeout (timers.js:436:11)
    at tryOnTimeout (timers.js:300:5)
    at listOnTimeout (timers.js:263:5)
    at Timer.processTimers (timers.js:223:10)
name: "ClusterClientNoResponseError"
pid: 12818
hostname: senge.local

